src/SFA.DAS.AdminService.Web/Controllers/Roatp/Apply/RoatpGatewayPeopleInControlController.cs

```diff
@@ -39,15 +39,50 @@ public async Task<IActionResult> GetGatewayPeopleInControlPage(Guid applicationI
             [HttpPost("/Roatp/Gateway/{applicationId}/Page/PeopleInControl")]
             public async Task<IActionResult> EvaluatePeopleInControlPage(PeopleInControlPageViewModel viewModel)
             {
+                var validationCheck = await GatewayValidator.Validate(viewModel);
+
+                if (validationCheck.Errors == null || !validationCheck.Errors.Any())
+                    return await SubmitGatewayPageAnswer(viewModel, $"{GatewayViewsLocation}/PeopleInControl.cshtml",
```

Comment:
I don't think this should call Submit when it knows it's just going to fail.

Reply:
It isn't, it's calling it when there aren't any errors

Reply:
Thanks, turns out I can't read. 😁

Reply:
Oh I think I got what you mean, you mean after it rebuilds the model. Ok.

Reply:
It's a bit clearer now cos it's dropping into a View directly rather than indirectly (via the submit answer business)